content_sync.info.yml

```diff
@@ -2,7 +2,7 @@ name: 'Content Sync'
 type: module
 description: 'Allows to import/export content using YAML files'
 package: Custom
-core: 8.x
```

Comment:
Should... this be being removed? It was left intact on some of the others...

... looks like it may only make sense of Drupal >= 8.8?... which is to say, yeah, makes sense here.